app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -730,6 +731,7 @@ class BrowserTabViewModel(
     fun onFireproofWebsiteSnackbarUndoClicked(fireproofWebsiteEntity: FireproofWebsiteEntity) {
         viewModelScope.launch {
             withContext(dispatchers.io()) {
+                pixel.fire(PixelName.FIREPROOF_WEBSITE_UNDO)
```

Comment:
The same as before, I was registering the user action more than the success. But the same, I don't have a strong reason to do not move it after delete executed. 

I guess moving the pixel 1 below the delete should work, as any error triggered from there should cancel the coroutine block.

Reply:
I didn't mean to change dispatchers. My comment was that for the same action (firing a pixel) sometimes it's called form inside the coroutine block and sometimes from outside. When sent from a coroutine block it is called from `io` or from `main` so that makes 3 different ways to call the same method. Choose one of them, and stick to it so when someone else reads the code there are no questions about how to execute the action